qaevents.md

```diff
@@ -0,0 +1,116 @@
+## Main Endpoint
+**GET /api/integration/qaevents**
```

Comment:
Somewhere we should describe at a high level what a "QA Event" is and where it comes from.   So, I'd expect there to be a summary section at the top which describes the concept...similar to how we have a summary section for the ORCID endpoints (see `orcidequeues.md` for example)